festivalApp/src/main/java/com/revature/festivalapp/pojos/FestivalEvent.java

```diff
@@ -23,27 +25,29 @@
 	private String eventLocation;
 	
 	@Column(name="event_start_date")
+	@Convert(converter=LocalDateAttributeConverter.class)
 	private LocalDate startDate;
 	
 	@Column(name="event_end_date")
+	@Convert(converter=LocalDateAttributeConverter.class)
 	private LocalDate endDate;
 	
 	@Column(name="event_desc")
 	private String eventDescription;
 	
-	@ManyToOne(cascade=CascadeType.ALL, fetch=FetchType.LAZY)
-	@JoinColumn(name="manager_email")
-	private Manager manager;
+	@ManyToOne(cascade= {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH}, 
+				fetch=FetchType.LAZY)
+	@JoinColumn(name="manager_id")
+	private User manager;
 
 	public FestivalEvent() {
 		super();
 		// TODO Auto-generated constructor stub
 	}
 
-	public FestivalEvent(Integer eventNumer, Integer audieceCapacity, String eventName, String eventLocation,
-			LocalDate startDate, LocalDate endDate, String eventDescription, Manager manager) {
+	public FestivalEvent(Integer audieceCapacity, String eventName, String eventLocation,
```

Comment:
audienceCapacity is misspelled, no big deal at this point I believe, unless we need to reference it later and you spell it right there

Reply:
Good catch. Thanks!